app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+  include Course::AchievementConditionalConcern
```

Comment:
You can just place it in `Course::Condition::AchievementsController`? Then there's no need to write it twice, that's just inheritance?